libraries/botbuilder-ai-orchestrator/api-extractor.json

```diff
@@ -0,0 +1,375 @@
+/**
+ * Config file for API Extractor.  For more info, please visit: https://api-extractor.com
+ */
+ {
+  "$schema": "https://developer.microsoft.com/json-schemas/api-extractor/v7/api-extractor.schema.json",
+
+  /**
+   * Optionally specifies another JSON config file that this file extends from.  This provides a way for
+   * standard settings to be shared across multiple projects.
+   *
+   * If the path starts with "./" or "../", the path is resolved relative to the folder of the file that contains
+   * the "extends" field.  Otherwise, the first path segment is interpreted as an NPM package name, and will be
+   * resolved using NodeJS require().
+   *
+   * SUPPORTED TOKENS: none
+   * DEFAULT VALUE: ""
+   */
+  // "extends": "./shared/api-extractor-base.json"
```

Comment:
It looks like we can do `api-extractor --config ../../api-extractor.json` and have a single shared config file.

Reply:
Unfortunately, [`mainEntryPointFilePath`](https://api-extractor.com/pages/configs/api-extractor_json/#mainentrypointfilepath), a required property in `api-extractor.json`, can't seem to look at the right locations:

![image](https://user-images.githubusercontent.com/40401643/118876934-aeb14d00-b8a2-11eb-9a43-2a2e6c6cd502.png)
![image](https://user-images.githubusercontent.com/40401643/118876974-ba9d0f00-b8a2-11eb-8832-3a325f30d777.png)

I wasn't able to get any of the "supported tokens" to work, nor does prepending with ".".

> The path is resolved relative to the folder of the config file that contains the setting; to change this, prepend a folder token such as <projectFolder>.

[Rushstack](https://github.com/microsoft/rushstack) seems to have one `api-extractor.json` for each app, too.

However, [I've made the base file and change the others to `extend` from it](https://github.com/microsoft/botbuilder-js/pull/3691/commits/f5fd6422045974bc6aa596b1d44a46387073bc7d).

Reply:
Ah - annoying. The "extends" method is a reasonable approach though. Thanks!